src/Catty/Firebase/FirebaseAnalyticsReporter.swift

```diff
@@ -81,4 +83,10 @@ class FirebaseAnalyticsReporter {
         }
         return ""
     }
+
+    @objc func formulaSaved(notification: Notification) {
+        let formula_string = getObjectClassName(for: notification)
+
+        analytics.logEvent("formula_saved", parameters: [AnalyticsParameterItemName: formula_string])
```

Comment:
This will always log the parameter `Formula`. Please use the display string, just like you did at the `CrashlyticsReporter`.